go.mod

```diff
@@ -89,3 +91,6 @@ require (
 	xorm.io/core v0.7.3
 	xorm.io/xorm v0.8.1
 )
+
+// Override xorm's outdated go-mssqldb dependency
```

Comment:
Is there an issue in xorm for this?  If not can we make one and then leave the issue as a comment here (or reference existing one).

Then there is a link to see when we might be able to remove this.

Reply:
@kylebrandt The problem is that we are on an old version on xorm, and if we upgrade to their latest it will most likely break for us.

Reply:
Ah, can you add that to the comment then? append 'since we are on an old version of xorm and not ready to upgrade' - and that it is about getting a newer version of thrift?

Basically the information future people will need to know if they ask themselves "Can I remove this?"

Reply:
Yeah thanks for the feedback @kylebrandt, I can do that. Just have to deal with more urgent stuff.

Reply:
Could we maybe move replace directive above require directive? Makes it easier to spot/not miss it has a replace directive.